src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -299,6 +346,24 @@ public ColumnMask getMask()
         return mask;
     }
 
+    @Nonnull
+    public ColumnConstraints getColumnConstraints()
+    {
+        return columnConstraints; // either Noop or actual constraints
```

Comment:
The comment is not true. `null` value is assigned to `columnConstraints` for example at line#158.